domains/api/http/transport.go

```diff
@@ -90,11 +90,11 @@ func MakeHandler(svc domains.Service, authn authn.Authentication, mux *chi.Mux,
 		r.Route("/{domainID}/invitations", func(r chi.Router) {
 			r.Use(api.AuthenticateMiddleware(authn, true))
 			r.Post("/", otelhttp.NewHandler(kithttp.NewServer(
-				sendInvitationEndpoint(svc),
+				sendDomainInvitationEndpoint(svc),
 				decodeSendInvitationReq,
 				api.EncodeResponse,
 				opts...,
-			), "send_invitation").ServeHTTP)
+			), "send_domain_invitation").ServeHTTP)
 			r.Get("/", otelhttp.NewHandler(kithttp.NewServer(
```

Comment:
Please rename `decodeListInvitationsReq` to `decodeListDomainInvitationsReq` and its related Response struct